charts/internal/shoot-system-components/charts/csi-driver-manila/values.yaml

```diff
@@ -84,10 +84,10 @@ csimanila:
         }
       }
 
-  topologyAwarenessEnabled: false
+  topologyAwarenessEnabled: true
   # Availability zone for each node. topologyAwarenessEnabled must be set to true for this option to have any effect.
   # If your Kubernetes cluster runs atop of Nova and want to use Nova AZs as AZs for the nodes of the cluster, uncomment the line below:
-  #nodeAZ: "$(curl http://169.254.169.254/openstack/latest/meta_data.json | jq -r .availability_zone)"
+  nodeAZ: "$(curl http://169.254.169.254/openstack/latest/meta_data.json | jq -r .availability_zone)"
```

Comment:
probably this needs to be moved directly into the `csi-driver-manila/templates/daemonset-node.yaml`?